app/poro/trails_list.rb

```diff
@@ -0,0 +1,18 @@
+class TrailsList
+  attr_reader :id, :location, :forecast, :trails
+
+  def initialize(city_state)
+    @id = nil
```

Comment:
don't really like having to add a useless id but fast_jsonapi was designed for database models, i suppose, and not for the odd goofy app like this one